### PR TITLE
1.Update 更新测试用例。

### PR DESCRIPTION
**Pull request by npk191954**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

config模块的测试用例滞后，运行出现错误，及时更新。

## Brief changelog

config模块的测试用例。

### CODE REVIEW

File: config/src/test/java/com/alibaba/nacos/config/server/controller/ConfigOpsControllerTest.java [hunk 1]
```diff
@@ -57,52 +57,51 @@
 @ContextConfiguration(classes = MockServletContext.class)
 @WebAppConfiguration
 public class ConfigOpsControllerTest {
-    
+
```
Comment (KomachiSion) [inline config/src/test/java/com/alibaba/nacos/config/server/controller/ConfigOpsControllerTest.java:60]: Please use nacos code style to reformat codes

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: config/src/test/java/com/alibaba/nacos/config/server/controller/ConfigOpsControllerTest.java (hunk 1 above)
Comment (npk191954) [reply config/src/test/java/com/alibaba/nacos/config/server/controller/ConfigOpsControllerTest.java]: It's done.

**Review by KomachiSion — APPROVED** [review]

**Comment by npk191954** [comment]
> checkstyle can't pass, need fix them first.
I have reformated them using nacos-code-style-for-idea.xml.

**Comment by npk191954** [comment]
> checkstyle can't pass, need fix them first.


I have reformated them using nacos-code-style-for-idea.xml.

**Comment by KomachiSion** [comment]
@npk191954 There are some conflicts with others PR.

**Pull request closed without merging**